async-nats/src/lib.rs

```diff
@@ -140,6 +142,7 @@ mod options;
 
 use crate::options::CallbackArg1;
 pub use client::Client;
+pub use client::PublishError;
```

Comment:
Could collapse to a single line

```suggestion
pub use client::PublishError;
```